src/base/check.h

```diff
@@ -37,18 +37,10 @@
 #include <ostream>
 
 #include "base/exception.h"
-#include "cvc4_export.h"
+#include "cvc5_export.h"
 
 // Define CVC5_NO_RETURN macro replacement for [[noreturn]].
-#if defined(SWIG)
-#define CVC5_NO_RETURN
-// SWIG does not yet support [[noreturn]] so emit nothing instead.
-#else
 #define CVC5_NO_RETURN [[noreturn]]
```

Comment:
This define is used a single time, and that is on `~FatalStream()`. Is it worth having a define?

Reply:
I think we can remove it.